public/js/pimcore/element/helpers/gridColumnConfig.js

```diff
@@ -456,7 +456,7 @@ pimcore.element.helpers.gridColumnConfig = {
             title: title,
             items: [formPanel],
             bodyStyle: "background: #fff;",
-            width: formPanel.items.items[0].width + 25,
+            width: 650,
```

Comment:
Haven't tested but something like 
```suggestion
            width: (formPanel.items?.items[0]?.width ?? 625) + 25,
```
?

Reply:
@kingjia90 `formPanel.items.items[0].width` is undefined. Regardless of whether a width is set or not.

Reply:
Ah ok, then why it was changed to this in https://github.com/pimcore/admin-ui-classic-bundle/pull/681 
@mattamon could you please have a second look on this?

Reply:
@kingjia90 Ah, there is a many to many field used. I will check this. I only test it with many to one.

Reply:
@kingjia90 I changed it.